ccloud-metricbeat-elastic-kibana/assets/metricbeat/modules.d/prometheus.yml

```diff
@@ -0,0 +1,74 @@
+# Module: prometheus
+# Docs: https://www.elastic.co/guide/en/beats/metricbeat/8.4/metricbeat-module-prometheus.html
+
+- module: prometheus
+  period: 900s
+  #hosts: ['https://api.telemetry.confluent.cloud/v2/metrics/cloud/export?resource.kafka.id=lkc-9kg320&resource.kafka.id=lkc-w9ykm&resource.schema_registry.id=lsrc-x1gxg']
```

Comment:
Can you add a comment what you've to put the params in the `hosts` instead of using params for future tracking reason

Reply:
updated comment